pdsfile/pds3file/__init__.py

```
@@ -11,6 +11,7 @@
 from pdsfile.pdsfile import PdsFile
 from . import rules
 from pdsfile.preload_and_cache import cache_lifetime_for_class
+import re
```

Flake8 (7.2.0)

[error] redefinition of unused 're' from line 6

(F811)